testsuite/features/step_definitions/common_steps.rb

```diff
@@ -1210,3 +1210,7 @@ def server_secret
   puts "URL: #{url}"
   step %(I enter "#{url}" as "calendar-data-text")
 end
+
+Then(/^text field "([^\"]*)" should contain "([^\"]*)"$/) do |field, text|
```

Comment:
```suggestion
Then(/^the text field "([^\"]*)" should contain "([^\"]*)"$/) do |field, text|
```

Reply:
I wonder if that duplicates the functionality of:
``` 
Then(/^I should see a "([^"]*)" text in element "([^"]*)"$/) do |text, element|
  within(:xpath, "//div[@id=\"#{element}\" or @class=\"#{element}\"]") do
    raise "Text #{text} not found in #{element}" unless has_content?(text)
  end
end
```

Reply:
Too bad, already merged, i'll fix the duplication in a separate PR